src/dbt/kipptaf/models/surveys/intermediate/int_surveys__survey_responses.sql

```diff
@@ -47,3 +52,84 @@ left join
     {{ ref("stg_reporting__terms") }} as rt
     on rt.name = fr.info_title
     and date(fr.last_submitted_time) between rt.start_date and rt.end_date
+
+union all
+
+select
+    safe_cast(sr.survey_id as string) as survey_id,
+    sr.survey_title as survey_title,
+    safe_cast(sr.response_id as string) as survey_response_id,
+    coalesce(
+        sr.response_string_value,
+        sr.response_map_value,
+        sr.response_option_value,
+        sr.response_rank_value
+    ) as answer,
+    sr.question_title_english as question_title,
+    sr.question_short_name as question_shortname,
+    concat(
+        sr.survey_link_default, '?snc=', sr.response_session_id, '&sg_navigate=start'
+    ) as survey_response_link,
+
+    regexp_extract(sr.campaign_name, r'\s(.*)') as survey_code,
+    'SURVEY' as survey_type,
+    sr.campaign_fiscal_year - 1 as academic_year,
+
+    eh.employee_number,
+    eh.preferred_name_lastfirst as respondent_name,
+    eh.management_position_indicator as is_manager,
+    eh.department_home_name as respondent_department_name,
+    eh.business_unit_home_name as respondent_legal_entity_name,
+    eh.report_to_preferred_name_lastfirst as respondent_manager_name,
+    eh.job_title as respondent_primary_job,
+    eh.home_work_location_name as respondent_primary_site,
+    eh.race_ethnicity_reporting,
+    eh.gender_identity as gender,
+    eh.mail,
+    eh.report_to_preferred_name_lastfirst as manager_name,
+    eh.report_to_mail as manager_email,
+    eh.report_to_user_principal_name as manager_user_principal_name,
+    eh.alumni_status,
+    eh.community_grew_up,
+    eh.community_professional_exp,
+    eh.level_of_education,
+    eh.primary_grade_level_taught,
+
+    timestamp(sr.response_date_started) as date_started,
+    timestamp(sr.response_date_submitted) as date_submitted,
+    safe_cast(
+        coalesce(
+            sr.response_string_value,
+            sr.response_map_value,
+            sr.response_option_value,
+            sr.response_rank_value
+        ) as numeric
+    ) as answer_value,
+    case
+        when
+            safe_cast(
+                coalesce(
+                    sr.response_string_value,
+                    sr.response_map_value,
+                    sr.response_option_value,
+                    sr.response_rank_value
+                ) as integer
+            )
+            is null
+        then 1
+        else 0
+    end as is_open_ended,
+from {{ ref("base_alchemer__survey_results") }} as sr
+left join
+    {{ ref("stg_reporting__terms") }} as rt
+    on rt.name = sr.survey_title
+    and date(sr.response_date_submitted) between rt.start_date and rt.end_date
```

Comment:
use refactored calc

Reply:
missed this one

Reply:
@cristinabaldor still need this date transformation refactored to `base_alchemer__survey_results`

Reply:
ok one more try